### ISSUE TITLE
Feature request: Plot, but don't break circuit

### ISSUE THREAD

**Issue by feschmidt** [mgely/qucat#31]
When omitting junctions or inductances, qucat returns a `ValueError: There should be at least one junction or inductor in the circuit`. It would be nice if the corresponding circuit could still be plotted, in case you wanted to just plot circuits

**Comment by gsteele13** [comment]
i agree, but i would suggest it's not 100% top priority, and there are also other circuit plotting libraries:

https://schemdraw.readthedocs.io/en/latest/

Although, this has no gui...

(good project idea: a jupyter widget gui that could be used with both qucat and schemdraw...)
